libs/live-config/jest.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  testPathIgnorePatterns: ["lib/", "lib-es/"],
+};
```

Comment:
This should allow you to get rid of the extra "@jest/globals" imports in `libs/live-config/src/FeatureFlags/useFeature.test.tsx`

```suggestion
module.exports = {
  preset: "ts-jest",
  testEnvironment: "node",
  testPathIgnorePatterns: ["lib/", "lib-es/"],
  globals: {
    "ts-jest": {
      isolatedModules: true,
    },
  },
};
```